### PR TITLE
Pulumi + Defang + Aiven Example

### PR DESCRIPTION
**Pull request by raphaeltm**

### CODE REVIEW

File: samples/nodejs/Remix Aiven Postgres/pulumi/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "defang-pulumi-remix-postgres",
+    "main": "index.ts",
+    "devDependencies": {
+        "@types/node": "^16"
+    },
+    "dependencies": {
+        "@defang-io/pulumi-defang": "^0.11.8",
```
Comment (lionello) [inline samples/nodejs/Remix Aiven Postgres/pulumi/package.json:8]: please test with latest `^0.15.1`

File: samples/nodejs/Remix Aiven Postgres/remix/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:20-alpine
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN npm install
```
Comment (lionello) [inline samples/nodejs/Remix Aiven Postgres/remix/Dockerfile:7]: usually `npm ci` (if you have a lock file)

**Review by lionello — CHANGES_REQUESTED** [review]

File: samples/nodejs/Remix Aiven Postgres/remix/Dockerfile (hunk 2 above)
Comment (raphaeltm) [reply samples/nodejs/Remix Aiven Postgres/remix/Dockerfile]: Updated.

File: samples/nodejs/Remix Aiven Postgres/pulumi/package.json (hunk 1 above)
Comment (raphaeltm) [reply samples/nodejs/Remix Aiven Postgres/pulumi/package.json]: Tested.

**Review by lionello — APPROVED** [review]

**Pull request merged**